File: Selenium/Activities/Activity_2.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;


public class Activity_2 {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("The tile of page is: " + driver.getTitle());
        driver.findElement(By.name("Username")).sendKeys("admin");
        driver.findElement(By.name("Password")).sendKeys("password");
        driver.findElement(By.xpath("//button[@class='ui button']")).click();
        Thread.sleep(2000);
        driver.quit();
    }
}

File: Selenium/Activities/Activity_2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print(f"Print of title is {driver.title}")
    driver.find_element(By.NAME,"Username").send_keys("admin")
    driver.find_element(By.NAME,"Password").send_keys("password")
    driver.find_element(By.XPATH,"//button[@class='ui button']").click()
    time.sleep(2)
    driver.quit()